src/cmd/cluster_intel_collector/version_collector.go

```diff
@@ -0,0 +1,25 @@
+package cluster_intel_collector
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"github.com/Dynatrace/dynatrace-operator/src/version"
+)
+
+const versionFileName = "operator-version.txt"
+
+func collectOperatorVersion(_ *intelCollectorContext, tarball *intelTarball) error {
```

Comment:
why have an `intelCollectorContext` parameter if it is not used?

Reply:
Because there's a common function definition for all collector functions which is used in main.